src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -26,6 +26,12 @@ private static string GetFileNameFromResourceName(string s)
 
         public int CreateEmptyProject(string languageName, string templateDir)
         {
+            if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "project.json")))
+            {
+                Reporter.Error.WriteLine($"Creating new {languageName} project failed, project.json already exists.");
```

Comment:
The second check, if there are files in the folder. Also, there should be a message that says that the projects already exists in the local folder and shouldn't mention `project.json` for any reason.